source/users.rst

```diff
@@ -177,6 +242,7 @@ Request:
     "username": "example",
     "password": "password",
     "email": "example@example.com"
+    "roles": ["intern"],
```

Comment:
"A role consists...a machine-readable slug, which will be passed to all ``POST`` endpoints."

Maybe I should be clearer, but the idea is that when data is going from the client to the server, it uses slugs in order to ensure there are no errors in transmission and that the data is easily readable, and when data is going from the server to the client, it uses the actual names in order to allow the client to display them to the user and have them be human-readable. This is the same way the rest of our API works.